app/views/propertyDetails/propertyDetailsHasBeenRevalued.scala.html

```diff
@@ -0,0 +1,85 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ApplicationConfig
+@import views.ViewUtils.titleBuilder
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.Implicits.RichRadios
+@import play.twirl.api.HtmlFormat
+@import _root_.utils._
+
+@this(newMain: newMain,
+        govukBackLink : GovukBackLink,
+        govukRadios : GovukRadios,
+        govukButton : GovukButton,
+        govukErrorSummary : GovukErrorSummary,
+        formHelper: FormWithCSRF)
+@(
+        propertyKey: String,
+        periodKey: Int,
+        backLink: Option[String],
+        mode: Option[String],
+        propertyDetailsHasBeenRevaluedForm: Form[HasBeenRevalued],
+        serviceInfoContent: Html = HtmlFormat.empty
+)(
+        implicit authContext: StandardAuthRetrievals,
+        messages: Messages,
+        request: Request[AnyContent],
+        appConfig: ApplicationConfig
+)
+
+@newMain(titleBuilder(messages("ated.property-details-value.hasBeenRevalued.title"), Some(propertyDetailsHasBeenRevaluedForm)), serviceInfoContent = serviceInfoContent) {
+
+ @if(backLink.isDefined) {
+  @govukBackLink(BackLink(
+   href = backLink.get,
+   content = Text("Back")
+  ))
+ }
+
+ @if(propertyDetailsHasBeenRevaluedForm.hasErrors) {
+  @govukErrorSummary(ErrorSummary().withFormErrorsAsText(propertyDetailsHasBeenRevaluedForm))
+ }
+
+ @formHelper(action = controllers.propertyDetails.routes.PropertyDetailsHasBeenRevaluedController.save(propertyKey, periodKey, mode)) {
+
+  @govukRadios(
+   Radios(
+    items = Seq(
+     RadioItem(
+      content = Text(messages("ated.property-details-value.yes")),
+      value = Some("true")
+     ),
+     RadioItem(
+      content = Text(messages("ated.property-details-value.no")),
+      value = Some("false")
+     )
+    ),
+    classes = "govuk-radios--inline"
+   ).withFormField(propertyDetailsHasBeenRevaluedForm("isPropertyRevalued"))
+   .withHeadingAndSectionCaption(
```

Comment:
Sorry to just dropping a comment here - Just realised there has been a accessibility change for screen reader related to caption and heading. Here is how it needs to be done - https://design.tax.service.gov.uk/hmrc-design-patterns/page-heading/ - caption needs to be h2 now instead on being inside of H1 as a span. Russel has already created a PR for this change, we can take a reference there.